client/components/autocomplete_list/index.coffee

```diff
@@ -10,7 +10,7 @@ request = require 'superagent'
 module.exports = (el, props) ->
   ReactDOM.render React.createElement(AutocompleteList, props), el
 
-module.exports.AutocompleteList = AutocompleteList = React.createClass
+module.exports = AutocompleteList = React.createClass
```

Comment:
Does this work on the Channel edit too? Just concerned that the first module.export is being overridden by the second one. I think we can remove the first one and add that line directly in Channel edit if it needs it.